cli/azd/pkg/environment/manager.go

```diff
@@ -325,8 +337,7 @@ func (m *manager) ensureValidEnvironmentName(ctx context.Context, spec *Spec) er
 
 			This value is typically used by the infrastructure as code templates to name the resource group that contains
 			the infrastructure for your application and to generate a unique suffix that is applied to resources to prevent
-			naming collisions.`),
-			DefaultValue: spec.Suggest,
```

Comment:
Just checking - was it intentional that we removed `DefaultValue: spec.Suggest` here?

Reply:
Likely a rebase issue if not intended since there was some refactoring in this area.